src/static/css/base.css

```diff
@@ -8,11 +8,21 @@
 	url('../fonts/ArialRoundedMTBold.svg#svgFontName') format('svg'); /* Legacy iOS */
 }
 
+@font-face {
+	font-family: "Eurostile";
+	src: url("../fonts/Eurostile.ttf");
+}
+
 body {
+	overflow-y: scroll !important;
```

Comment:
If there's anyway in the future we can figure out how to do this without needing to use `!important` that would be tight hahaha.